src/type-layout.md

```diff
@@ -264,13 +264,23 @@ For all other enumerations, the layout is unspecified.
 
 Likewise, combining two primitive representations together is unspecified.
 
-### The `packed` Representation
+### `align(x)` Representation
+
+The `align(x)` representation can be used on `struct`s and `union`s to raise the
```

Comment:
The paragraph structure here should mirror the packed representation section. So which categories of types it can be applied to should be one paragraph, and what it does should be another.